juju/juju
# show/list k8s controllers display relevant info

## Description of change

Modify show/list controllers/models to correctly display k8s info. ie k8s models have no machines. Instead we display nodes and unit count.

## QA steps

bootstrap lxd and l8s controllers
add iaas and caas models
run various CLI commands to check output
